client/src/components/Feed/Post.js

```diff
@@ -741,6 +756,17 @@ const Post = ({
               )}
               <div className="pre-header">
                 <span>{t(`feed.${objective}`)}&nbsp;&nbsp;•</span>
+                <span>
+                  &nbsp;&nbsp;
+                  {t(
+                    workMode && workMode !== "both"
```

Comment:
This logic seems repeated several times, maybe extract it out

Reply:
Done